### PR TITLE
[plugin.video.vrt.nu] 2.2.1

### PR DESCRIPTION
**Pull request by dagwieers**
### Description
The new version includes some cosmetic changes, it has breadcrumbs
support, various fixes to credentials and tokens, a new metadata engine
and new artwork.

This PR replaces #2482 and fixes one review remark in that PR.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.vrt.nu/README.md [hunk 1]
```diff
@@ -49,6 +49,25 @@ You can report issues at [our GitHub project](https://github.com/pietje666/plugi
 you can send a message to [our Facebook page](https://facebook.com/kodivrtnu/).
 
 ## Releases
+### v2.2.0 (2019-08-16)
```
Comment (mediaminister) [inline plugin.video.vrt.nu/README.md]: This should be 2.2.1!

Comment (dagwieers) [reply plugin.video.vrt.nu/README.md]: Arghl, thanks !

**Comment by dagwieers** [comment]
@basrieter Your concerns were addressed, please re-review :-)

**Comment by basrieter** [comment]
Wil do.

File: plugin.video.vrt.nu/addon.xml [hunk 2]
```diff
@@ -48,39 +48,56 @@ Met deze Kodi add-on kan je alle video streams van VRT NU bekijken.
         <platform>all</platform>
         <license>GNU General Public License, v3</license>
         <news>
+v2.2.1 (2019-08-16)
```
Comment (enen92) [inline plugin.video.vrt.nu/addon.xml]: Keep this only for the current version

Comment (basrieter) [reply plugin.video.vrt.nu/addon.xml]: @dagwieers Please keep this in mind for the next version.

Comment (dagwieers) [reply plugin.video.vrt.nu/addon.xml]: We like to keep more than one entry in the changelog, and especially in this case where v2.2.0 was not released due to this review process.

What is the reason for limiting the "news" entry to 1500 characters and just one entry?

**Pull request merged**